src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Excluded attributes corresponding to `LogRecord.EventId`,
+`LogRecord.CategoryName` and `LogRecord.Exception` from the exported data. This
```

Comment:
Something to consider:
Removing Exception could be undone. Agree its experimental, but 1.0 (of Trace) onwards, we have done this already: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/Trace/ActivityExtensions.cs#L86 despite the convention being experimental! 

Would be good at checking if Java is doing this in stable or as experimental only! (Either way, this is lower priority than the goal is shipping stable OTLP, so this is no way a blocking comment.)

Reply:
It is doubtful that Java has this to deal with in their exporter given their data model does not include an exception field.

With instrumentation Java is well organized in managing expectations utilizing semver appropriately with regards to experimental conventions. We definitely do not want to couple the ongoing stabilization of conventions to the potential need of a major version bump of our OTLP exporter.